data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/network_action/InjectionSetpointImpl.java

```diff
@@ -7,33 +7,30 @@
 
 package com.farao_community.farao.data.crac_impl.remedial_action.network_action;
 
+import com.farao_community.farao.data.crac_api.ElementaryAction;
 import com.farao_community.farao.data.crac_api.NetworkElement;
-import com.farao_community.farao.data.crac_api.usage_rule.UsageRule;
 import com.fasterxml.jackson.annotation.JsonTypeName;
 import com.powsybl.iidm.network.*;
 import org.apache.commons.lang3.NotImplementedException;
 
-import java.util.List;
-
 /**
  * Injection setpoint remedial action: set a load or generator at a given value.
  *
  * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
  */
 @JsonTypeName("injection-setpoint")
-public final class InjectionSetpoint extends AbstractSetpointElementaryNetworkAction {
+public final class InjectionSetpointImpl implements ElementaryAction {
```

Comment:
I think it is a good opportunity to override equals and hashcode in these classes

Reply:
done